1-exercises/A-array-find/exercise.js

```diff
@@ -3,7 +3,10 @@
   Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-// write your code here
+function longName(name) {
```

Comment:
Please check this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
https://www.w3schools.com/jsref/jsref_find.asp
https://www.youtube.com/watch?v=8SkHWeDoTf0
You are going to find the correct syntax and how to use find method